2020-12-26 08:36:41.0393 INFO This is a normal notification
2020-12-26 08:36:41.0817 DEBUG This comes from debug
2020-12-26 08:36:41.0817 WARN This is a yellow warning
2020-12-26 08:36:41.0817 ERROR Something wrong happens
2020-12-26 08:39:52.1749 INFO This is a normal notification
2020-12-26 08:39:52.1749 DEBUG This comes from debug
2020-12-26 08:39:52.1749 WARN This is a yellow warning
2020-12-26 08:39:52.1749 ERROR Something wrong happens
2020-12-26 08:41:30.0130 INFO This is a normal notification
2020-12-26 08:41:30.0391 DEBUG This comes from debug
2020-12-26 08:41:30.0391 WARN This is a yellow warning
2020-12-26 08:41:30.0391 ERROR Something wrong happens
2020-12-26 08:44:46.6497 INFO This is a normal notification
2020-12-26 08:44:46.6813 DEBUG This comes from debug
2020-12-26 08:44:46.6813 WARN This is a yellow warning
2020-12-26 08:44:46.6813 ERROR Something wrong happens
2020-12-26 08:46:16.3124 INFO This is a normal notification
2020-12-26 08:46:16.3358 DEBUG This comes from debug
2020-12-26 08:46:16.3358 WARN This is a yellow warning
2020-12-26 08:46:16.3358 ERROR Something wrong happens
2020-12-26 08:46:45.3635 INFO This is a normal notification
2020-12-26 08:46:45.3723 DEBUG This comes from debug
2020-12-26 08:46:45.3723 WARN This is a yellow warning
2020-12-26 08:46:45.3723 ERROR Something wrong happens
2020-12-26 08:49:48.4146 INFO This is a normal notification
2020-12-26 08:49:48.4461 DEBUG This comes from debug
2020-12-26 08:49:48.4461 WARN This is a yellow warning
2020-12-26 08:49:48.4461 ERROR Something wrong happens
2020-12-26 09:08:48.4138 INFO This is a normal notification
2020-12-26 09:08:48.4465 DEBUG This comes from debug
2020-12-26 09:08:48.4465 WARN This is a yellow warning
2020-12-26 09:08:48.4465 ERROR Something wrong happens
2020-12-26 09:31:13.7068 INFO This is a normal notification
2020-12-26 09:31:13.7536 DEBUG This comes from debug
2020-12-26 09:31:13.7536 WARN This is a yellow warning
2020-12-26 09:31:13.7536 ERROR Something wrong happens
2020-12-26 10:19:01.4718 INFO This is a normal notification
2020-12-26 10:19:01.5219 DEBUG This comes from debug
2020-12-26 10:19:01.5219 WARN This is a yellow warning
2020-12-26 10:19:01.5219 ERROR Something wrong happens
2020-12-26 10:19:11.7448 ERROR Something went wrong in the GetCompanies action AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Object -> IEnumerable`1
System.Object -> System.Collections.Generic.IEnumerable`1[[Entities.DataTransferObjects.CompanyDto, Entities, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]
 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Company -> CompanyDto
Entities.Models.Company -> Entities.DataTransferObjects.CompanyDto
   at lambda_method(Closure , Company , CompanyDto , ResolutionContext )
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\workspace\C#\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 35
2020-12-26 10:22:05.8503 INFO This is a normal notification
2020-12-26 10:22:05.8780 DEBUG This comes from debug
2020-12-26 10:22:05.8780 WARN This is a yellow warning
2020-12-26 10:22:05.8780 ERROR Something wrong happens
2020-12-26 10:28:40.7675 INFO This is a normal notification
2020-12-26 10:28:40.7998 DEBUG This comes from debug
2020-12-26 10:28:40.7998 WARN This is a yellow warning
2020-12-26 10:28:40.7998 ERROR Something wrong happens
2020-12-26 10:29:02.4473 INFO This is a normal notification
2020-12-26 10:29:02.4715 DEBUG This comes from debug
2020-12-26 10:29:02.4715 WARN This is a yellow warning
2020-12-26 10:29:02.4715 ERROR Something wrong happens
2020-12-26 10:30:13.1713 INFO This is a normal notification
2020-12-26 10:30:13.1901 DEBUG This comes from debug
2020-12-26 10:30:13.1901 WARN This is a yellow warning
2020-12-26 10:30:13.1901 ERROR Something wrong happens
2020-12-26 10:31:06.7081 INFO This is a normal notification
2020-12-26 10:31:06.7352 DEBUG This comes from debug
2020-12-26 10:31:06.7352 WARN This is a yellow warning
2020-12-26 10:31:06.7352 ERROR Something wrong happens
2020-12-26 10:32:19.1885 INFO This is a normal notification
2020-12-26 10:32:19.2539 DEBUG This comes from debug
2020-12-26 10:32:19.2539 WARN This is a yellow warning
2020-12-26 10:32:19.2539 ERROR Something wrong happens
2020-12-26 10:32:19.3356 INFO This is a normal notification
2020-12-26 10:32:19.3356 DEBUG This comes from debug
2020-12-26 10:32:19.3356 WARN This is a yellow warning
2020-12-26 10:32:19.3432 ERROR Something wrong happens
2020-12-26 10:32:38.4381 INFO This is a normal notification
2020-12-26 10:32:38.4683 DEBUG This comes from debug
2020-12-26 10:32:38.4683 WARN This is a yellow warning
2020-12-26 10:32:38.4683 ERROR Something wrong happens
2020-12-26 10:33:03.1256 INFO This is a normal notification
2020-12-26 10:33:03.1417 DEBUG This comes from debug
2020-12-26 10:33:03.1417 WARN This is a yellow warning
2020-12-26 10:33:03.1417 ERROR Something wrong happens
2020-12-26 11:26:30.8438 INFO This is a normal notification
2020-12-26 11:26:30.8588 DEBUG This comes from debug
2020-12-26 11:26:30.8588 WARN This is a yellow warning
2020-12-26 11:26:30.8588 ERROR Something wrong happens
2020-12-26 11:27:08.7582 INFO This is a normal notification
2020-12-26 11:27:08.7832 DEBUG This comes from debug
2020-12-26 11:27:08.7832 WARN This is a yellow warning
2020-12-26 11:27:08.7832 ERROR Something wrong happens
2020-12-26 11:27:27.9929 ERROR Something went wrong in the GetCompanies action System.Exception: This exception is created intentionally
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\workspace\C#\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 33
2020-12-26 11:30:00.5343 INFO This is a normal notification
2020-12-26 11:30:00.5613 DEBUG This comes from debug
2020-12-26 11:30:00.5613 WARN This is a yellow warning
2020-12-26 11:30:00.5613 ERROR Something wrong happens
2020-12-26 11:30:22.6247 INFO This is a normal notification
2020-12-26 11:30:22.6454 DEBUG This comes from debug
2020-12-26 11:30:22.6454 WARN This is a yellow warning
2020-12-26 11:30:22.6555 ERROR Something wrong happens
2020-12-26 11:30:35.1396 INFO This is a normal notification
2020-12-26 11:30:35.1722 DEBUG This comes from debug
2020-12-26 11:30:35.1722 WARN This is a yellow warning
2020-12-26 11:30:35.1722 ERROR Something wrong happens
2020-12-26 11:46:01.4912 INFO This is a normal notification
2020-12-26 11:46:01.5162 DEBUG This comes from debug
2020-12-26 11:46:01.5162 WARN This is a yellow warning
2020-12-26 11:46:01.5162 ERROR Something wrong happens
2020-12-26 11:46:11.0021 ERROR Something went wrong: System.Exception: This exception is created intentionally
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\workspace\C#\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-12-26 13:49:40.4205 ERROR Something went wrong: System.Exception: This exception is created intentionally
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\workspace\C#\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-12-26 13:50:08.1016 INFO This is a normal notification
2020-12-26 13:50:08.1016 DEBUG This comes from debug
2020-12-26 13:50:08.1016 WARN This is a yellow warning
2020-12-26 13:50:08.1016 ERROR Something wrong happens
2020-12-26 13:50:33.5548 INFO This is a normal notification
2020-12-26 13:50:33.5800 DEBUG This comes from debug
2020-12-26 13:50:33.5800 WARN This is a yellow warning
2020-12-26 13:50:33.5800 ERROR Something wrong happens
2020-12-26 13:50:39.2491 ERROR Something went wrong: System.Exception: This exception is created intentionally
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\workspace\C#\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-12-26 14:07:18.3311 INFO Company with id 3d490a70-94ce-4d15-9494-5248280c2ce4 does not exist in database.
2020-12-26 15:23:48.2409 INFO Company with id c9d4c053-49b6-410c-bc78-2d54a999187a does not exist.
2020-12-26 15:24:14.1964 INFO Company with id c9d4c053-49b6-410c-bc78-2d54a9991871 does not exist.
2020-12-26 15:27:03.3809 INFO Employee with id 86dba8c0-d178-41e7-938c-ed49778fb52b does not exist
2020-12-26 15:27:27.3486 INFO Company with id c9d4c053-49b6-410c-bc78-2d54a9991871 does not exist.
2020-12-26 16:06:20.2344 INFO Company with id 3d490a70-94ce-4d15-9494-5248280c2ce4 does not exist in database.
